lib/src/kbasesearchengine/KBaseSearchEngineServer.java

```diff
@@ -85,6 +85,10 @@ private void quietLoggers() {
         ((Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME))
                 .setLevel(Level.INFO);
     }
+    private String getUserName(final AuthToken authPart){
```

Comment:
blank line

Reply:
what?

Reply:
*blank line*

Reply:
what do you mean blank line

Reply:
Add a blank line here, between the functions